client/src/components/comment-panel/comment-panel.js

```diff
@@ -0,0 +1,135 @@
+import React from 'react'
+import { connect } from 'react-redux'
+// Helpers
+import InlineSVG from 'svg-inline-react'
+import classNames from 'classnames/bind'
+import { shadowDocument } from '../../shadowDomHelper'
+import Comments from '../comments/comments'
+// Styles
+import commentPanelStyles from './comment-panel.scss'
+// Assets
+import CloseIcon from '../../assets/svg/baseline-close-24px.svg'
+
+const css = classNames.bind(commentPanelStyles)
+
+const mapStateToProps = (state) => {
+  const users = (state.persist || {}).users || {}
```

Comment:
Tätäkö ei siis tarvii?

Reply:
@bqqbarbhg miks state.persist || {} tarvitaan, eihän {}.users kuitenkaa toimi? Miks ei vaa (state.persist).users || {}

Reply:
If `state.persist` is `undefined` then `state.persist.users` resolves to `undefined.users` which will throw an error. However if we replace `state.persist` with `{}` it will resolve to `{}.users` which does _not_ throw an error but returns `undefined`.

This is pretty janky though so maybe we should use [`R.path()`](https://ramdajs.com/docs/#path)